### ISSUE TITLE
Node cannot find module 'gonzales-pe'

### ISSUE THREAD

**Issue by tbassetto** [tonyganch/gonzales-pe#245]
I have followed the instructions in the README to install parser as a project dependency:
```
npm install --save git://github.com/tonyganch/gonzales-pe.git#dev
```
And I am using it like this:
```js
const fs = require('fs');
const gonzales = require('gonzales-pe');

const content = fs.readFileSync('app.15b3fb9686901533f471.css', 'utf-8');
const parseTree = gonzales.parse(content);
```
But I get:
```
Error: Cannot find module 'gonzales-pe'
```
I suspect it's because `package.json` has `"./lib/gonzales"` for the `"main"` property, which doesn't exist!

How to use the parser in a node project then?

**Comment by DanPurdy** [comment]
Because you installed it directly from Github I believe you'll have to build it before you can use the package. Might not be correct but you can try moving into the gonzales-pe folder and running 

`npm run build` 

see [here](https://github.com/tonyganch/gonzales-pe/blob/dev/package.json#L20) and then try using the package again.

**Comment by tbassetto** [comment]
Hey. I have installed the package again using `npm install --save gonzales-pe` and it works as expected. Is it an official package published on npm? If yes it should probably used in the README instead of recommending the installation via GitHub.

**Comment by DanPurdy** [comment]
Yes it is it's just that there is a build step involved. Before publishing to NPM a build step is invoked and that output is pushed to NPM whereas obvously targeting Github means that pre-publish hasn't happened.

I agree that it's probably a good idea to document this in the Readme though.

**Comment by tonyganch** [comment]
Hey guys! There was a `postinstall` script that allowed installing straight from GitHub without any additional build steps, but there was a problem reported on Windows platform and I disabled it:
https://github.com/tonyganch/gonzales-pe/commit/f3357117329a8c477ced52686e74a7a9dc7cb9b5

I completely forgot about it since then and never restored it back. I'll take a look after v4.0 release.

**Comment by tonyganch** [comment]
Related issue: #149

**Comment by tonyganch** [comment]
@tbassetto, could you please test installing `gonzales-pe` from GitHub once again? I've added `postinstall` script, tested it locally and both Travis and Appveyor look okay, but I'm not 100% sure about it.

**Comment by DanPurdy** [comment]
If it's a purely bash script it won't be suitable.. Bash might work on the latest windows but for legacy it will fail.. If we want to keep it pure we should look at node postinstall scripts

**Comment by bgriffith** [comment]
Talking of which - after updating the dep to v4 getting;

```
> gonzales-pe@4.0.0 postinstall /home/travis/build/sasstools/sass-lint/node_modules/gonzales-pe
> bash ./scripts/postinstall.sh
bash: ./scripts/postinstall.sh: No such file or directory
```

https://travis-ci.org/sasstools/sass-lint/jobs/176886355

**Comment by tonyganch** [comment]
Thanks for the news, guys. I've removed the script, released 4.0.1 without it and will just update docs, so we will live without GitHub installations.

**Comment by jdalton** [comment]
I can change this to a [`prepare`](https://docs.npmjs.com/misc/scripts#prepublish-and-prepare) script so that `gonzales-pe` will work for `git:` installs.
